app/com/lucidchart/open/cashy/utils/DownloadHelper.scala

```diff
@@ -0,0 +1,28 @@
+package com.lucidchart.open.cashy.utils
+
+import org.apache.http.client.methods.{HttpGet, HttpPost}
+import org.apache.http.entity.StringEntity
+import org.apache.http.impl.client.HttpClientBuilder
+
+case class DownloadFailedException(message: String) extends Exception(message)
+
+object DownloadHelper extends DownloadHelper
+class DownloadHelper {
+
+  def downloadBytes(url: String): Array[Byte] = {
+    val httpClient = HttpClientBuilder.create().build()
+
+    val httpGet = new HttpGet(url)
+    val response = httpClient.execute(httpGet)
+
+    val statusCode = response.getStatusLine().getStatusCode()
+    if (statusCode < 200 || statusCode >= 300) {
```

Comment:
We should probably just check if it's a 200. Other 20x responses don't really make sense as successes when you're trying to download something.